src/PositionContext.ts

```diff
@@ -172,8 +172,8 @@ export class PositionContext {
         return this._hoverInfo;
     }
 
-    // tslint:disable-next-line:cyclomatic-complexity // Grandfathered in
-    public get completionItems(): Promise<Completion.Item[]> {
+    // tslint:disable-next-line:cyclomatic-complexity max-func-body-length // Grandfathered in
+    public async getCompletionItems(): Promise<Completion.Item[]> {
         if (this._completionItems === undefined) {
             this._completionItems = Promise.resolve([]);
```

Comment:
I'm not getting auto-completions for `resourceGroup()` anymore. I did a little digging and think it's related to my above comment and this line. You're basically calling `getCompletionItems` twice, which means the first sets this to an empty array here and the second just returns that empty array before the first call finishes getting the completions. I don't think there was a risk of race conditions before when this was synchronous, but I think now you have to be careful about that.

Reply:
Weird that the tests passed.

Reply:
Tests passed because they were unit tests, calling the get directly.  Added testing.